01_Data/src/layers/sequelize/repository/Reservation.ts

```diff
@@ -62,11 +62,16 @@ export class SequelizeReservationRepository extends SequelizeRepository<Reservat
           evseId: evseDBId,
           idToken: reserveNowRequest.idToken,
           groupIdToken: reserveNowRequest.groupIdToken ?? null,
+          isActive: isActive === true ? isActive : false,
```

Comment:
are you sure about this change? Seems a bit off to me that if I leave out an optional, it will still filter by that optional being false. I would expect to have the optional ignored in the query.